src/inspector/models/notation/lines/pedalsettingsmodel.h

```diff
@@ -50,6 +50,7 @@ class PedalSettingsModel : public TextLineSettingsModel
     void setLineType(int newType);
 
     PropertyItem* m_lineType = nullptr;
+    bool rosetteHookSelected;
```

Comment:
Better to always default-initialise all bool/int/double/char/… values:
```suggestion
    bool rosetteHookSelected = false;
```